src/script/printer/traced_object.h

```diff
@@ -0,0 +1,410 @@
+/*
```

Comment:
Can you move this to the public headers (include/...)? Because it's used by the actual printer functions, which could be outside of libtvm.